data-raw/dev/korea.R

```diff
@@ -6,7 +6,6 @@
 #' @importFrom rvest html_nodes html_text
```

Comment:
if we are going to support these in the future we should put on a branch. Otherwise can probably just drop.